webcompat/static/js/lib/bugform.js

```diff
@@ -550,7 +558,6 @@ function BugForm() {
             "The image is too big! Please choose something smaller than 4MB.";
           wcEvents.trigger("flash:error", { message: msg, timeout: 5000 });
         }
-        this.loadingIndicator.hide();
       }, this)
```

Comment:
We still need to handle the error case, otherwise it will continue to pulse forever.

![bug](https://user-images.githubusercontent.com/67283/37739230-7ee8989e-2d27-11e8-8422-12cf644759c8.gif)